src/main/docker/entrypoint.sh

```diff
@@ -66,10 +67,37 @@ loadApiParameters() {
   fi
 
   if [ -z $CYFACE_API_ENDPOINT ]; then
-    CYFACE_API_ENDPOINT="/api/v3/"
+    CYFACE_API_ENDPOINT="/api/v4/"
   fi
 }
 
+loadKeycloakParameters() {
+  if [ -z "$CYFACE_KEYCLOAK_CALLBACK" ]; then
+    # FIXME: only use http if this stays internal (localhost)
```

Comment:
FIXME: Add to the staging deployment task to see what the callback URL actually has to look like.

Reply:
As explained above. Do avoid using the Term Keycloak here. Any OpenID Identity Provider will work.